openedx/core/djangoapps/enrollments/serializers.py

```diff
@@ -77,14 +80,48 @@ class CourseEnrollmentSerializer(serializers.ModelSerializer):
     """
     course_details = CourseSerializer(source="course_overview")
     user = serializers.SerializerMethodField('get_username')
+    finished = serializers.SerializerMethodField()
```

Comment:
We should probably also specify the method name as is done for the user field.

Reply:
@kaizoku we can't do this because there's a quality error raised:
```AssertionError: It is redundant to specify `get_finished` on SerializerMethodField 'finished' in serializer 'CourseEnrollmentSerializer', because it is the same as the default method name. Remove the `method_name` argument.```

Reply:
Ah gotcha, my mistake.